### PR TITLE
Jetpack Manage: More explanation about the manage feature.

### PR DESCRIPTION
**Pull request by enejb**

### CODE REVIEW

File: modules/module-info.php [hunk 1]
```diff
@@ -957,3 +957,25 @@ function jetpack_site_icon_more_link() {
...
+function jetpack_custom_jetpack_manage() { ?>
+
+
+	<div class="jp-info-img">
+		<img class="jp-info-img" src="<?php echo plugins_url( basename( dirname( dirname( __FILE__ ) ) ) . '/images/screenshots/manage.png' ) ?>" alt="<?php esc_attr_e( 'Manage your site from WordPress.com', 'jetpack' ) ?>" width="300" height="150" />
+	</div>
+
+	<p><em><?php esc_html_e( 'Manage feature allows you to update your self-hosted WordPress sites and your WordPress.com sites from a single dashboard on WordPress.com.', 'jetpack' ); ?></em></p>
+	<p><?php printf( __( '<strong>Plugins</strong><br /> Now you can update plugins, set plugins to automatically update, and activate or deactivate plugins on a per-site basis or in bulk from <a href="%s">wordpress.com/plugins</a>.', 'jetpack' ), 'https://wordpress.com/plugins' ); ?></p>
+	<p><?php printf( __( '<strong>Posts and Pages</strong><br />Add new post or page to any of your sites from a single inteface.', 'jetpack' ) ); ?></p>
```
Comment (enejb) [inline modules/module-info.php]: Should be Add a new post or page.

File: modules/module-info.php [hunk 2]
```diff
@@ -957,3 +957,25 @@ function jetpack_site_icon_more_link() {
 }
 add_action( 'jetpack_learn_more_button_site-icon', 'jetpack_site_icon_more_link' );
 // Site Icon: STOP
+
+// Manage: Start
+function jetpack_custom_jetpack_manage() { ?>
+
+
+	<div class="jp-info-img">
+		<img class="jp-info-img" src="<?php echo plugins_url( basename( dirname( dirname( __FILE__ ) ) ) . '/images/screenshots/manage.png' ) ?>" alt="<?php esc_attr_e( 'Manage your site from WordPress.com', 'jetpack' ) ?>" width="300" height="150" />
```
Comment (beaulebens) [inline modules/module-info.php]: Could use something more descriptive like "Manage all of your WordPress sites, self-hosted or not, from WordPress.com"

File: modules/module-info.php [hunk 3]
```diff
@@ -957,3 +957,25 @@ function jetpack_site_icon_more_link() {
...
+	</div>
+
+	<p><em><?php esc_html_e( 'Manage feature allows you to update your self-hosted WordPress sites and your WordPress.com sites from a single dashboard on WordPress.com.', 'jetpack' ); ?></em></p>
```
Comment (beaulebens) [inline modules/module-info.php]: "Enabling Manage allows you to update your self-hosted WordPress sites along with any WordPress.com sites you have, all in one simple dashboard."

File: modules/module-info.php (hunk 1 above)
Comment (beaulebens) [inline modules/module-info.php]: I'd probably just change that whole statement to something like 

"Create and edit Posts and Pages on any of your sites using the streamlined editor on WordPress.com"

**Comment by samhotchkiss** [comment]
@enejb - this looks good to me, feel free to merge when you're happy with it

**Pull request merged**